src/charm.py

```diff
@@ -259,10 +510,23 @@ def _unit_name_to_pod_name(self, unit_name: str) -> str:
             unit_name: name in "postgresql-k8s/0" format.
 
         Returns:
-            pod name in "psotgresql-k8s-0" format.
+            pod name in "postgresql-k8s-0" format.
         """
         return unit_name.replace("/", "-")
 
+    def force_primary_change(self):
+        """Force primary changes immediately.
+
+        This function is needed to handle cases related to
```

Comment:
provide a 1 sentence summary as to why (and keep the link)

Reply:
Added more info on https://github.com/canonical/postgresql-k8s-operator/pull/9/commits/ee0018899c32c6f9b60debcc5179ef39eb6ac02c, This should be removed in the next PR as the problem that it's related too was solved in the current Juju version.